biom/table.py

```diff
@@ -3713,6 +3808,105 @@ def any_value(vals, id_, md):
 
         return t
 
+    def to_dataframe(self):
+        """Convert matrix data to a Pandas SparseDataFrame
+
+        Notes
+        -----
+        Metadata are not included.
+
+        Returns
+        -------
+        pd.SparseDataFrame
+            A SparseDataFrame indexed on the observation IDs, with the column
+            names as the sample IDs.
+
+        Examples
+        --------
+        >>> from biom import example_table
+        >>> df = example_table.to_dataframe()
+        >>> df
+             S1   S2   S3
+        O1  0.0  1.0  2.0
+        O2  3.0  4.0  5.0
+        """
+        # avoid dense conversion
+        # http://stackoverflow.com/a/17819427
+        mat = [pd.SparseSeries(v.toarray().ravel()) for v in self.matrix_data]
+        df = pd.SparseDataFrame(mat, index=self.ids(axis='observation'),
+                                columns=self.ids())
+        return df
+
+    def metadata_to_dataframe(self, axis):
```

Comment:
Should the default axis be observation? I think that's the most common use-case.